### ISSUE TITLE
Not all boards are visible in jira init

### ISSUE THREAD

**Issue by shyim** [ankitpokhrel/jira-cli#278]
**Describe the bug**
Not all boards are visibile in jira iinit

**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="0.3.0", GitCommit="ea53355d5b72994a5b17a63e6e800ce9d8a23f4e", CommitDate="2022-01-09T19:50:19+00:00", GoVersion="go1.17.5", Compiler="gc", Platform="darwin/arm64")
    ```
2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    cloud
    ```
3. What operating system are you using? Also mention version.
    ```
    macOs
    ```
4. What terminal are you using? Also mention version.
    ```
    iterm 2
    ```

**To Reproduce**

1. Have many jira boards
2. jira init
3. Your board is missing

**Expected behavior**

Board is listed there

**Screenshots**

**Additional context**

**Comment by ankitpokhrel** [comment]
@shyim the API returns 50 results by default. I never realized that a project would've this many boards TBH. As a workaround, you can manually edit the config to add board details. The whole board thing needs some rework it seems.

**Comment by ankitpokhrel** [comment]
Hi @shyim, this should be fixed with #322 which adds search option when adding board during `jira init`.

![Screen Shot 2022-03-08 at 11 02 55 AM](https://user-images.githubusercontent.com/2364546/157218086-8057dc69-6571-4c9e-a16f-3290b2927547.png)

Please feel free to comment or open a new issue if this doesn't work for you. Thanks!
